src/commands/maven_assert_expression.yml

```diff
@@ -0,0 +1,23 @@
+description: >
+  Checks that some expression evaluated from the Maven project matches with a certain pattern.
+  This job will succeed if the expression contains the pattern, and fail otherwise.
+  Thus it can be used to allow/block flows depending on the value of the expression.
+
+parameters:
+  expression:
+    description: 'An expression to evaluate. Note that this must not include the surrounding ${...}'
```

Comment:
which surrounding ${...} 🤔 

this might be confusing unless you provide a bit of context